<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLook</class>
 <widget class="QWidget" name="WidgetLook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>云直播-观众</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/new/prefix1/logo_R.ico</normaloff>:/new/prefix1/logo_R.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>直播公告：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_public">
         <property name="text">
          <string>欢迎使用云直播</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>主播房间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>当前房间数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_room_num">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1,1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>当前房间：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_room_info">
             <property name="text">
              <string>房间号：主播名：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_look">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;黑体&quot;;
background-color: rgb(161, 161, 161)
</string>
           </property>
           <property name="text">
            <string>主播正赶来的路上！</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>鲜花</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>麦克风</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_3">
               <property name="text">
                <string>666</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_4">
               <property name="text">
                <string>飞机</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_5">
               <property name="text">
                <string>赞赞赞</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_6">
               <property name="text">
                <string>巧克力</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>赠送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>打开弹幕</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_msg">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_send">
               <property name="text">
                <string>发送</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="chat">
            <attribute name="title">
             <string>msg</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QTextEdit" name="te_msg"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="member">
            <attribute name="title">
             <string>member</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTextEdit" name="te_mem"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>当前房间人数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
